cmd/aem/crypto.go

```diff
@@ -81,11 +81,16 @@ func (c *CLI) cryptoProtectCmd() *cobra.Command {
 				c.Error(err)
 				return
 			}
-			c.SetOutput("value", protectedValue)
+			c.SetOutput("protected", protectedValue)
```

Comment:
I prefer naming style `topic_trait` not `protectedValue` but `valueProtected` because then vars are grouped logically / the same topic has the same string prefix;

how about `valueProtected` here; or just value as it was designed?

Reply:
For me valueProtected is also fine, just I thought to use same convention as in crypto response, just to make it consistent

Reply:
you should follow own conventions instead of being influenced by convention coming from outside to make your tool users confused without a clear reason